translations/en.json

```diff
@@ -1330,7 +1332,7 @@
     "add-seed-account": "Add account with a seed phrase",
     "account-exists-title": "Account already exists",
     "add-private-key-account": "Add account from private key",
-    "user-not-found": "User not found",
+    "user-not-found": "Profile not found",
```

Comment:
I want also to change the key to `profile-not-found`, but I would also need to change the key on all the translation files. Let me know if I can proceed with that!

Reply:
You just need to change the `en.json` file. We use [Lokalise](https://lokalise.com/) to manage translations. 

When you update an existing translation, just inform @jinhojang6 about it.

Reply:
Just rename it here and message @jinhojang6.